framework/functions/optimization-recommendations.md

```diff
@@ -20,6 +20,7 @@ In addition, organizations often find it challenging to actually execute or acti
 -	Right type/purpose – the type of resource is selected so that there is minimum overhead across all attributes of the resource (for example across CPU, memory and disk). Avoid using a generic resource for specialty tasks.
 -	Right number – dynamically add/remove resources within groups (such as behind load balancers or through pools of compute) as demand fluctuates.
 -	Service selection – Switch to managed services or serverless options that inherently perform the scaling.
+- Right Level of use - Identify resources or services which are being used for intended purpose but may be overpriced based on the value received. (https://www.linkedin.com/pulse/who-pays-watchmen-understand-your-azure-monitor-costs-jan-karstens)
```

Comment:
```suggestion
- Right Level of use - Identify resources or services which are being used for intended purpose but may be overpriced based on the value received. See [this article for examples](https://www.linkedin.com/pulse/who-pays-watchmen-understand-your-azure-monitor-costs-jan-karstens).
```

Reply:
@robmartin33 What do you think of this suggestion? That way the link isn't just hanging at the end of the bullet point, but is part of the overall thought. Let me know what you think! Otherwise looks good to go.